### ISSUE TITLE
Bug: Light/Dark mode toggle focus indicator

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#5420]
# Summary

From Twitter:
Light/Dark mode toggle focus indicator

## Where did you spot the bug?

All pages

## What is the problem?

There is no focus indicator to indicate that the light/dark mode toggle has focus

## What did you expect to happen?

There should be a clear focus indicator when the light/dark mode toggle has focus

## Steps to reproduce the issue

1. Open any page in the browser
2. Start pressing the tab key
3. It is clear where the focus is until you tab out of the last menu item
4. It looks like the focus is lost but,
5. if you tab again, the focus will be indicated on the search icon
